load_confounds/confounds.py

```diff
@@ -11,6 +11,85 @@
 import json
 
 
+def _check_params(confounds_raw, params):
+    """Check that specified parameters can be found in the confounds."""
+    not_found_params = []
+    for par in params:
+        if not par in confounds_raw.columns:
+            not_found_params.append(par)
+    if not_found_params:
+        raise MissingConfound(params=not_found_params)
+    return None
+
+
+def _find_confounds(confounds_raw, keywords):
+    """Find confounds that contain certain keywords."""
+    list_confounds = []
+    missing_keys = []
+    for key in keywords:
+        key_found = False
+        for col in confounds_raw.columns:
+            if key in col:
+                list_confounds.append(col)
+                key_found = True
+        if not key_found:
+            missing_keys.append(key)
+    if missing_keys:
+        raise MissingConfound(keywords=missing_keys)
+    return list_confounds
```

Comment:
Not sure if this will make the climate better:
```python
def _find_confounds(confounds_raw, keywords):
    """Find confounds that contain certain keywords."""
    list_confounds = []
    missing_keys = []
    for key in keywords:
        key_found = [col for col in confounds_raw.columns if key in col]
        if not key_found:
            missing_keys.append(key)
        else:
            list_confounds.extend(key_found)
    if missing_keys:
        raise MissingConfound(keywords=missing_keys)
    return list_confounds
```

Reply:
I have incorporated your suggested fix, and it works! THANKS